docs/playbook/create-glue-jobs.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Creating Glue jobs
+description: "Creating Glue jobs in terraform"
+layout: playbook_js
+tags: playbook
+---
+
+## Creating Glue jobs using the Glue jobs Terraform template
+
+#### 1. Copy an existing Glue job or use the example below and populate the variables
+If the file `23-aws-glue-jobs-<YOUR_DEPARTMENT_NAME>.tf` in the terraform directory exists, copy an existing module block. 
+
+- Do not change any spacing before the `=`
+- Do not remove the `"quotation marks"` around any values which use them
+
+_Create a new tf file for your department, named `23-aws-glue-jobs-<YOUR_DEPARTMENT_NAME>.tf` if it does not exist and refer to the [example](#example-module-block) below_
+
+Update the `module` name in the format `your-unique-module-name`, and unique to all other module names in this file (it is helpful to keep the same naming convention as your dataset/folder)
+
+#### The following input variables are required:
+- __department__: This will be `module.department_<YOUR_DEPARTMENT_NAME>`
+- __job_name__: The name of your glue job prefixed with your department name e.g. "housing repairs address cleaning"
+- __glue_scripts_bucket_id__: `module.glue_scripts.bucket_id`
+
+#### The following variables are optional:
+- __job_description__: A description of the AWS glue job e.g. "Glue job description"
+- __job_parameters__: Parameters used in your Glue job. For example the source database and table
+    - If you are setting __job_parameters__, you should include the below parameter:
+      - ```
+        "--TempDir" = "${module.glue_temp_storage.bucket_url}/${module.department_housing_repairs.identifier}/"
```

Comment:
If we want everyone to include this can we just accept the temp bucket ID as a variable and get the module to add this as a variable? It feels simpler to me

Reply:
Yes sounds like a good idea